.github/workflows/release_update_docs_px_dev.yaml

```diff
@@ -0,0 +1,101 @@
+---
+name: update-docs-px-dev-on-release
+on:
+  push:
```

Comment:
should remove this `push`

Reply:
ah, I missed removing that. Updated in latest revision.